pkg/vendir/fetch/githubrelease/sync.go

```diff
@@ -165,17 +210,70 @@ func (d Sync) matchesAssetName(name string) (bool, error) {
 	return false, nil
 }
 
+func (d Sync) fetchTagSelection() (string, error) {
+	opt := &github.RepositoryListByOrgOptions{
+		ListOptions: github.ListOptions{PerPage: 10},
+	}
+
+	tags := []string{}
+	for {
+		tagList, resp, err := d.client.Repositories.ListTags(context.Background(), strings.Split(d.opts.Slug, "/")[0], strings.Split(d.opts.Slug, "/")[1], &github.ListOptions{})
```

Comment:
lets pull out `strings.Split(d.opts.Slug, "/")[0]` and `strings.Split(d.opts.Slug, "/")[1]` as vars outside of the loop with names like ownerName, repoName.